src/runtime/runtime.cs

```diff
@@ -371,6 +393,12 @@ private static void InitializePlatformData()
             IntPtr platformModule = PyImport_ImportModule("platform");
             IntPtr emptyTuple = PyTuple_New(0);
 
+            fn = PyObject_GetAttrString(platformModule, "system");
```

Comment:
Why are we not in NETSTANDARD? Shouldn't we have RuntimeInformation available? Then we wouldn't need any of this code.

Reply:
I don't think we are (yet).

But if we're going to have defines, better factor it out as a function so we don't need to query python. We're not going to run a version compiled for Linux on macOS anyways.